### PR TITLE
Update team-settings version in Helm chart [skip ci]

### PR DESCRIPTION
**Pull request by zebot**
Image tag: `4.3.0-v0.28.28-a2f11cf`
Release: [`v4.3.0`](https://github.com/wireapp/wire-team-settings/releases/tag/v4.3.0)

### CODE REVIEW

File: charts/team-settings/values.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ resources:
     cpu: "1"
 image:
   repository: quay.io/wire/team-settings
-  tag: "4.2.0-v0.28.28-1e2ef7"
+  tag: "4.3.0-v0.28.29-0-a2f11cf"
```
Comment (jschaul) [inline charts/team-settings/values.yaml:12]: @Yserz the automatic PR made here uses a tag that doesn't exist on quay.io:

I can see these last versions on https://quay.io/repository/wire/team-settings?tab=tags;

```
production
4.2.0-v0.28.28-f58c700
staging
4.2.0-v0.28.28-8f89a7e
...
```

Can something be done on your side with respect to align this versioning?

**Review by jschaul — CHANGES_REQUESTED** [review]

File: charts/team-settings/values.yaml (hunk 1 above)
Comment (smatting) [reply charts/team-settings/values.yaml:12]: Has this issue been fixed by https://github.com/wireapp/wire-team-settings/pull/5037 ? If so can you please retrigger a new PR @Yserz ?

File: changelog.d/0-release-notes/team-settings-upgrade [hunk 2]
```diff
@@ -0,0 +1 @@
+Upgrade team-settings version to 4.3.0-v0.28.29-0-a2f11cf
```
Comment (julialongtin) [inline changelog.d/0-release-notes/team-settings-upgrade:1]:
```suggestion
Upgrade team-settings version to 4.3.0-v0.28.28-0-a2f11cf
```

File: charts/team-settings/values.yaml (hunk 1 above)
Comment (julialongtin) [inline charts/team-settings/values.yaml:12]:
```suggestion
  tag: "4.3.0-v0.28.28-0-a2f11cf"
```

Comment (julialongtin) [inline charts/team-settings/values.yaml]:
```suggestion
  tag: "4.3.0-v0.28.28-0-a2f11cf"
```

**Comment by smatting** [comment]
Changed to 4.3.0-v0.28.28-a2f11cf, which is the only 4.3.0 tag on quay

**Review by smatting — APPROVED** [review]

**Pull request merged**